### PR TITLE
Improve and expand conventions

### PR DESCRIPTION
**Pull request by glyn**
Help contributors by clarifying the preferred conventions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,10 +34,17 @@ change.
...
 ### Conventions
 
-Basic conventions around source files formatting are captured in the `.editorconfig` file.
-Many editors support that file natively. Others (such as VS code) require a plugin, see https://editorconfig.org/.
+Each entry is of the form:
+```
+* [The Literal Title of The Book](https://a-link-to-the/book/) - an optional comment if the title isn't self-explanatory
+```
+
+the `.editorconfig` file captures basic source files formatting conventions.
```
Comment (glyn) [inline README.md:42]:
```suggestion
The `.editorconfig` file captures basic source files formatting conventions.
```

**Review by glyn — COMMENTED** [review]
Fix a typo.

**Comment by glyn** [comment]
@kw-fn if you are happy, please feel free to merge.

**Comment by kw-fn** [comment]
Look good to me! Thanks!

**Pull request merged**